localenv/telemetry/grafana/provisioning/dashboards/example.json

```diff
@@ -1592,6 +1643,7 @@
           "sort": "none"
         }
       },
+      "pluginVersion": "11.3.1",
```

Comment:
Not part of your changes, but can you remove the `* 0.5` from L1654? Since we are summing by source we don't need to halve it to account for having both Cloud nine and happy life bank instances running